src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1515,11 +1515,12 @@ def create_vm(vm_name, resource_group_name, image=None, size='Standard_DS1_v2',
               private_ip_address=None, public_ip_address=None, public_ip_address_allocation='dynamic',
               public_ip_address_dns_name=None, os_disk_name=None, os_type=None, storage_account=None, os_caching=None,
               data_caching=None, storage_container_name=None, storage_sku=None, use_unmanaged_disk=False,
-              attach_os_disk=None, data_disk_sizes_gb=None, image_data_disks=None, vnet_name=None,
-              vnet_address_prefix='10.0.0.0/16', subnet=None, subnet_address_prefix='10.0.0.0/24', storage_profile=None,
-              os_publisher=None, os_offer=None, os_sku=None, os_version=None, storage_account_type=None, vnet_type=None,
-              nsg_type=None, public_ip_type=None, nic_type=None, validate=False, custom_data=None, secrets=None,
-              plan_name=None, plan_product=None, plan_publisher=None, license_type=None):
+              attach_os_disk=None, attach_data_disks=None, data_disk_sizes_gb=None, image_data_disks=None,
```

Comment:
`image_data_disks` is the data disks coming with the image itself. Most images only contain an os disk, but some images have data disks. BTW, this argument is hidden from users, our validation code figures it out for users.